frappe/public/js/frappe/views/workspace/workspace.js

```diff
@@ -90,6 +91,46 @@ frappe.views.Workspace = class Workspace {
 		}
 	}
 
+	prepare_new_and_edit() {
+		this.$page = $(`
+		<div class="editor-js-container"></div>
+		<div class="workspace-footer">
+			<button data-label="New" class="btn btn-default ellipsis btn-new-workspace">
+				<svg class="es-icon es-line icon-xs" style="" aria-hidden="true">
+					<use class="" href="#es-line-add"></use>
+				</svg>
+				<span class="hidden-xs" data-label="Edit">New</span>
+			</button>
+			<button class="btn btn-default btn-sm mr-2 btn-edit-workspace" data-label="Edit">
+				<svg class="es-icon es-line  icon-xs" style="" aria-hidden="true">
+					<use class="" href="#es-line-edit"></use>
+				</svg>
+				<span class="hidden-xs" data-label="Edit">
+					<span><span class="alt-underline">E</span>dit</span>
+				</span>
+			</button>
```

Comment:
Shouldn't the button labels be translated?

Reply:
https://github.com/frappe/frappe/pull/27256